format/Flight.proto

```diff
@@ -301,6 +301,36 @@ message Location {
  */
 message Ticket {
   bytes ticket = 1;
+  // Data representing some part of the data retrievable by the ticket.
+  //
+  // `inlined_completeness` indicates what part of the data retriavable
+  // by the ticket this represents. This is provided as an optimization for
+  // client/server applications that want to reduce latency to first result
+  // without requiring another RPC round-trip to retrieve the ticket.  applications
+  // built on top of Flight are responsible for any negotiation necessary on whether
+  // inlining data is appropriate.
+  //
+  // The size of inlined_data is expected to be small (typically less then 1MB) 
+  // and inlining too much data across tickets can run into underlying transport 
```

Comment:
Is there any hard-set number to be aware of here?
Seems important not to accidentally hit payload size limit in the transport

Reply:
We might already be overriding but gRPC has a default of 4MB limit.

Reply:
I'm hesitant to document this since there are experiments with other transports.

Reply:
Ah that is good to know, thank you. Will have to see how that can be configured because there are some cases in which a query may fetch ~10MB of data (or more)

Reply:
It's up to the transport. I believe we override this by default in C++ and Java, yeah, but I don't think we should document it here.